### DISCUSSION TITLE
Are the latest website fixes automatic?

### DISCUSSION THREAD

**Discussion by osfanbuff63** [darkreader/darkreader#9206] [Q&A] [answered]
A quick question, are the fixes for various websites (say, #9192) applied automatically, or are they released as part of the update system?

**Comment by Gusted** [comment, marked as answer]
Please check, https://github.com/darkreader/darkreader#site-fixes

**Comment by ghost** [comment]
man automatic site fixes not working! does it take time for it to sync? how can i manually update the site fixes for my chrome exntesion?
